spec/generator/target_spec.rb

```diff
@@ -34,7 +34,7 @@ module Generator
 
           target = double("Target").as_null_object
 
-          allow(target).to receive(:include_files).and_return('**/*.swift')
+          allow(target).to receive(:include_files).and_return(['**/*.swift'])
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.